src/sentry/static/sentry/app/views/settings/components/settingsNavigation.jsx

```diff
@@ -4,19 +4,36 @@ import React from 'react';
 
 import SettingsNavigationGroup from 'app/views/settings/components/settingsNavigationGroup';
 import SentryTypes from 'app/proptypes';
+import HookStore from 'app/stores/hookStore';
 
 class SettingsNavigation extends React.Component {
   static propTypes = {
     hooks: PropTypes.array,
     hookConfigs: PropTypes.array,
     navigationObjects: PropTypes.arrayOf(SentryTypes.NavigationObject).isRequired,
+    organization: SentryTypes.Organization,
   };
 
   static defaultProps = {
     hooks: [],
     hookConfigs: [],
   };
 
+  componentDidMount() {
+    let {organization} = this.props;
+    if (!organization) return;
+
+    let exposed = organization.features.includes('sso-paywall-experiment');
+    let data = {
```

Comment:
I think over here we want to log exposure only to the people who are IN the experiment. Quick n dirty way to do this is to return a ternary from the Experiment -- 0 for not exposed, 1 for exposed, 2 for not-applicable. Then log exposure only if you have a 0/1 value.

Reply:
we didn't have to worry about this before because it was implicit in the backend -- we would only log exposure if we invoked the experiment class. now that we're suppressing exposure logging, we need to deal with this manually. this would be a good thing to keep in mind when designing the frontend experiment abstraction -- a way to know if the user is even IN the experiment.